plugin/src/models/logger.ts

```diff
@@ -1,14 +1,32 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Do we still need these eslint-disable comments?

Reply:
good catch, no we do not because i addressed this in the config itself